edgelet/build/windows/util.ps1

```diff
@@ -10,7 +10,9 @@ function Test-RustUp
 
 function GetPrivateRustPath
 {
-    Join-Path -Path (Get-IotEdgeFolder) -ChildPath 'rust-windows-arm/rust-windows-arm/bin/'
+    $toolchain = Get-Content -Encoding UTF8 (Join-Path -Path (Get-EdgeletFolder) -ChildPath 'rust-toolchain')
```

Comment:
this file's changes are not related to this PR.

Reply:
Not it doesn't but I need it to build successfully for testing.